pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -112,11 +120,14 @@ def style_loss(
         layers = ("relu3_1", "relu4_1")
 
     if stride is None:
+        # https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/cnnmrf.lua#L53
```

Comment:
Could you re-check what the paper states? I remember asking myself if this is just an interpretation issue.

Reply:
In the paper it is described like this: ([paper](https://arxiv.org/pdf/1601.04589.pdf) second section chapter 5)
> To  achieve  the  best  synthesis  quality  we  set  the  stride  to one so patches are very densely sampled.

So `stride=1` should be correct.

Reply:
Yeah, I was thinking maybe he meant "overlap of one" which is `stride=2`. But since he explicitly used the word "stride" this leaves no room for interpretation.